src/js/common/schemaform/FormPage.jsx

```diff
@@ -19,12 +20,14 @@ import { setData } from './actions';
 
 const fields = {
   ObjectField,
-  ArrayField
+  ArrayField,
+  address: Address
 };
 
 const reviewFields = {
   ObjectField: ReviewObjectField,
-  ArrayField
+  ArrayField,
+  address: Address
```

Comment:
I think for now you could change this to `address: ReviewObjectField`. Otherwise, we're going to show form fields in "view" mode on the review page.

We may need to write a custom Address component for the review page, but the normal object field might be good enough.